core/src/main/scala/org/apache/spark/rdd/ReliableCheckpointRDD.scala

```diff
@@ -133,9 +134,13 @@ private[spark] object ReliableCheckpointRDD extends Logging {
     val broadcastedConf = sc.broadcast(
       new SerializableConfiguration(sc.hadoopConfiguration))
     // TODO: This is expensive because it computes the RDD again unnecessarily (SPARK-8582)
+    logInfo(s"The checkpoint compression codec is " +
```

Comment:
This should be logged if compression is enabled (none is not a supported compression codec).
It could also be rolled into the timing info log message below.